test/recipes/75-test_quicapi_data/ssltraceref.txt

```diff
@@ -79,10 +79,26 @@ Sent Packet
   Packet Number: 0x00000000
 Sent Datagram
   Length: 1200
```

Comment:
just a question: where do those changes coming from?

Reply:
The quicapi test has a subtest for confirming that the SSL_trace api works properly for them.  It does this by hook SSL_trace to trace a QUIC connection, which it then compares against a known trace (which I'm guessing was just recorded long ago).  With the addition of the retry request, the SSL_trace command records new SSL records (which include both the retry frame being sent, as well as additional transport parameters).  This alters the trace records, causing failures in the test.  These fixups bring the expected SSL trace records back into line with what we actually record.